releasenotes/notes/added-sign-feature-parameterexpression-57693dd69103dc8c.yaml

```diff
@@ -0,0 +1,22 @@
+---
+prelude: >
+    New feature to display sign of a Parameter
+features:
+  - |
+    Introduced a new feature that adds support for displaying the sign of the Parameter of the value of an expression of a Parameter class.
+    Instead of using numpy or other libraries, the user can use the instance of the ParameterExpression class to display the sign.
```

Comment:
```suggestion
    Introduced a new feature that adds support for calculating the sign of the Parameter of the value of an expression of a Parameter class.
    Instead of using numpy or other libraries, the user can use the instance of the ParameterExpression class to calculate the sign.
```